packages/loopring_v3/contracts/impl/libexchange/ExchangeWithdrawals.sol

```diff
@@ -450,25 +471,66 @@ library ExchangeWithdrawals
 
 
     // == Internal Functions ==
-    function withdrawTokens(
+
+    // The transfer can fail because of a transfer error or because the transfer uses
+    // more than GAS_LIMIT_SEND_TOKENS gas.
+    // The function returns true when successful, otherwise false.
+    // Works similar to address.send() for sending ETH.
+    function sendTokens(
         ExchangeData.State storage S,
-        address accountOwner,
+        address to,
         uint16  tokenID,
         uint    amount
         )
         internal
+        returns (bool)
     {
-        address payable owner = address(uint160(accountOwner));
+        // Limit the amount of gas that can be used
+        uint gasLimit = ExchangeData.GAS_LIMIT_SEND_TOKENS();
+        return sendTokensInternal(S, to, tokenID, amount, gasLimit);
+    }
+
+    // The transfer is guaranteed to succeed using as much gas as needed,
+    // otherwise it throws. Always returns true.
+    // Works similar to address.transfer() for transfering ETH.
+    function transferTokens(
```

Comment:
I would suggest to merge `sendTokens`, `transferTokens` and `sendTokensInternal ` as one function.

Reply:
Merged the functions into one.